src/linear_algebra/finite_dimensional.lean

```diff
@@ -807,6 +807,21 @@ begin
   { rw [h, findim_top, H] at this, exact findim_eq_zero.1 (add_right_injective _ this) }
 end
 
+lemma algebra_hom.bijective {F : Type*} [field F] {E : Type*} [field E] [algebra F E]
+  [finite_dimensional F E] (ϕ : E →ₐ[F] E) : function.bijective ϕ :=
+begin
+  have inj : function.injective ϕ.to_linear_map := ϕ.to_ring_hom.injective,
+  exact ⟨inj, (injective_iff_surjective_of_findim_eq_findim rfl).mp inj⟩,
+end
+
+/-- Biijection between algebra equivalences and algebra homomorphisms -/
+noncomputable def alg_equiv_equiv_alg_hom (F : Type u) [field F] (E : Type v) [field E]
+  [algebra F E] [finite_dimensional F E] : (E ≃ₐ[F] E) ≃ (E →ₐ[F] E) :=
+{ to_fun := λ ϕ, ϕ.to_alg_hom,
+  inv_fun := λ ϕ, alg_equiv.of_bijective ϕ (algebra_hom.bijective ϕ),
```

Comment:
```suggestion
  inv_fun := λ ϕ, alg_equiv.of_bijective ϕ (ϕ.bijective),
```

Reply:
I tried this, but it didn't work, maybe because `alg_hom.bijective` is in a `linear_map` namespace?

Reply:
Probably. So please move it to a better place.

Reply:
Ok, should be fixed now.